controllers/src/main/scala/cromwell/pipeline/controller/ProjectController.scala

```diff
@@ -19,13 +19,14 @@ class ProjectController(projectService: ProjectService)(
     path("projects") {
       concat(
         get {
-          parameter('name.as[String]) { name =>
-            onComplete(projectService.getUserProjectByName(name, accessToken.userId)) {
-              case Success(project)                         => complete(project)
-              case Failure(e: ProjectNotFoundException)     => complete(StatusCodes.NotFound, e.getMessage)
-              case Failure(e: ProjectAccessDeniedException) => complete(StatusCodes.Forbidden, e.getMessage)
-              case Failure(e)                               => complete(StatusCodes.InternalServerError, e.getMessage)
-            }
+          parameter('name.as[String]) {
+            name =>
+              onComplete(projectService.getUserProjectByName(name, accessToken.userId)) {
+                case Success(project)                         => complete(project)
+                case Failure(e: ProjectNotFoundException)     => complete(StatusCodes.NotFound, e.getMessage)
+                case Failure(e: ProjectAccessDeniedException) => complete(StatusCodes.Forbidden, e.getMessage)
+                case Failure(e)                               => complete(StatusCodes.InternalServerError, e.getMessage)
+              }
```

Comment:
Scalafmt config is different in team again, I suppose :)

Reply:
Build was failing until I changed it.
I have no idea why it worked fine before, because I get the same changes when I run `sbt fmt` on the develop branch.